org-roam.el

```diff
@@ -870,6 +870,26 @@ whose title is 'Index'."
         (concat (expand-file-name org-roam-directory) path)
       index)))
 
+;;;; dealing with file-wide properties
+(defun org-roam--set-global-prop (name value)
+  "Set a file property called NAME to VALUE.
+
+If the property is already set, it's value is replaced."
+  (save-excursion
+    (widen)
+    (goto-char (point-min))
+    (if (re-search-forward (concat "^#\\+" name ": \\(.*\\)") (point-max) t)
+        (replace-match (concat "#+" name ": " value) t)
```

Comment:
Fixed case here is required to avoid unwanted capitalisations.

```
#+ROAM_ALIAS: TBD
```

If you add an alias like 'lower', without fixed case it will be capitalised 😸